SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -2,6 +2,9 @@
 
 #include "SpatialGDKEditorToolbar.h"
 
+#include "SpatialConstants.cxx"
```

Comment:
Why no clang formatting around these? And why not include the header first?

Reply:
Clang reorders them, putting the .cxx before the .h. And the cxx file does not include the header. So order matters.
I did that deliberately so the cxx file does not look like something that would be easily usable.